### DISCUSSION TITLE
20220122: Volume Control Broken?

### DISCUSSION THREAD

**Discussion by findnix** [rern/rAudio#778] [Fixed Issues]
Hi @rern !

Since Update 20220122 the volume control shows no effect.
Slider is moving.
No volume change. Neither for "Mixer device" nor for "MPD software".
alasmixer is working correct.

```
# cat /etc/asound.conf
defaults.pcm.card 1
defaults.ctl.card 1

# aplay -l | grep ^card
card 0: Headphones [bcm2835 Headphones], device 0: bcm2835 Headphones [bcm2835 Headphones]
card 1: Digital [Pre Box S2 Digital], device 0: USB Audio [USB Audio]

# amixer scontrols
Simple mixer control 'Pre Box S2 Digital ',0
Simple mixer control 'Pre Box S2 Digital ',1
```

from the journal ...
`Jan 23 20:16:37 rAudio-VIE mpdidle.sh[8635]: amixer: Unable to find simple control 'Pre Box S2 Digital',0`

**Comment by rern** [comment]
Please try:
- Addons > rAudio 1 
- long-press Update
- OK for Branch UPDATE

**Reply by findnix** [reply]
THANK YOU !!
Ok now.
